### DISCUSSION TITLE
没有收到激活码

### DISCUSSION THREAD

**Discussion by warm3snow** [getlantern/lantern-forum-cn#622] [Q&A]
6.19，购买了一年的专业版，没有收到激活码

补：从6.19到11.22了，已经过去5个多月了。到现在没有收到激活码，也没有退款，奉劝各位千万不要被骗了！！！

**Comment by warm3snow** [comment]
没有人么？从昨天就发邮件给lantern，到现在没有人回复。

**Reply by MaxMKMax** [reply]
@warm3snow 
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号又想到这个链接. 谢谢. 
您发完后请自行删掉个您的个人资料和单据.

**Reply by warm3snow** [reply]
支付单据已发

**Reply by warm3snow** [reply]
没有收到退款是怎么回事？支付单据和账号都发了啊。问下多久能退款 @Mike777Khoo

**Reply by warm3snow** [reply]
为啥没有退款啊？这都一个月了。咋回事？

**Reply by warm3snow** [reply]
为啥没有退款啊？这都五个多月了。咋回事？

**Reply by MaxMKMax** [reply]
@warm3snow 您好. 请您重新提供我支付单据和账号邮箱. 在这里提供给我. 我帮您重新提交退款.

**Reply by warm3snow** [reply]
> @warm3snow 您好. 请您重新提供我支付单据和账号邮箱. 在这里提供给我. 我帮您重新提交退款.

能不能负责的帮忙解决一下问题

支付单据：
![IMG_A44E1BD8CB74-1](https://github.com/user-attachments/assets/6eb37d70-1ced-43dc-8a24-fcfa26b475e9)

账号邮箱：xueyanghan@hotmail.com

**Comment by Jianliang-ouc** [comment]
我也遇到了同样的问题！

**Reply by MaxMKMax** [reply]
@Jianliang-ouc https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号又想到这个链接. 谢谢. 
您发完后请自行删掉个您的个人资料和单据.
